src/main/java/com/toy/board/springbootboard/ApiResponse.java

```diff
@@ -0,0 +1,35 @@
+package com.toy.board.springbootboard;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class ApiResponse <T> {
+    private int statusCode;
+    private T data;
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd HH:mm:SS", timezone = "Aisa/Seoul")
+    private LocalDateTime serverDateTime;
+
+    public ApiResponse(int statusCode, T data) {
+        this.statusCode = statusCode;
+        this.data = data;
+        this.serverDateTime = LocalDateTime.now().truncatedTo(ChronoUnit.MILLIS);
+    }
+
+    public static <T> ApiResponse<T> ok(T data){
+        return new ApiResponse<>(200, data);
+    }
+
+    public static <T> ApiResponse<T> fail(int statusCode, T data){
+        return new ApiResponse<>(statusCode, data);
+    }
```

Comment:
정적팩토리로만 생성하는데 생성자를 열어놓을 이유가 있을까요?
모두 정적팩토리로만 객체를 생성할 수 있도록 유도해주세요 

```suggestion
    private ApiResponse(int statusCode, T data) {
        this.statusCode = statusCode;
        this.data = data;
        this.serverDateTime = LocalDateTime.now().truncatedTo(ChronoUnit.MILLIS);
    }

    public static <T> ApiResponse<T> ok(T data){
        return new ApiResponse<>(200, data);
    }

    public static <T> ApiResponse<T> fail(int statusCode, T data){
        return new ApiResponse<>(statusCode, data);
    }
```

Reply:
헉 자동완성 기능을 통해서 작성하다보니 오타가 난것같네요! 감사합니다.